compass/src/components/Applications/CreateApplicationModal/CreateApplicationModal.component.js

```diff
@@ -290,9 +279,6 @@ class CreateApplicationModal extends React.Component {
       if (scenariosQuery.labelDefinition) {
```

Comment:
this can become a one-liner: `const availableScenarios = scenariosQuery.labelDefinition ? JSON.parse(scenariosQuery.labelDefinition.schema).items.enum : []`

Also, JSON.parse() throws an error if the argument could not be parsed. So a good practice would be to try/catch that statement first and then use it.